.remarkrc

```diff
@@ -1,5 +0,0 @@
-{
```

Comment:
I want to lint markdown files.

Reply:
I believe we should use remark.
This project lives in the remark organization.
Other projects maintained by unified also use remark.

Reply:
It's OK to use `remark-lint`, but it's not enabled.

Reply:
Cool, good! I will work on docs next and want to add them there.
I kept it out for now to make it easier to review (same as https://github.com/mdx-js/eslint-mdx/pull/419#issuecomment-1229432001)